### ISSUE TITLE
 The size of the same input image gives different results even when keeping the width/height scale

### ISSUE THREAD

**Issue by NeRR0** [google/mediapipe#2473]
- OS Platform and Distribution: Ubuntu 20.04
- MediaPipe version: 0.8.7.1
- Solution: Pose (heavy model)
- Programming Language and version: Python 3.7 / 3.8

**Describe the expected behavior:**
The input image size should produce the same results (or close) as long as:
- its size is greater than or equal to the model input (256x256x3 - checked in Netron)
- the scale between the width and height of the image has been maintained at various sizes (e.g. 640x520 / 4519x3673)

so in our case the input photo with a size of 4519 x 3673 should give the same results (or very close) as the 640x520 photo
4519/640 = 7.0609
3673 / 7.0609 = 520.1886

4519/3673 = 1.2303
640/520 = 1.2308

**Standalone code you may have used to try to get what you need :**
Colab:
Long version: https://colab.research.google.com/drive/1qC2Ok9LZkRaDHdwsvTpeYsW3vQgq9jit?usp=sharing
Short version: https://colab.research.google.com/drive/1ot7ob7z-Fnjb3Kz1ndKonQekN6kbCDDa?usp=sharing

**Expected answer :**
Providing the ideal image size to obtain the highest possible accuracy of the model

**Comment by sgowroji** [comment]
Hi @NeRR0, Could you please check the same with the latest version and confirm the same ?
